shared-images/openresty-nginx/nginx.vh.default.conf

```diff
@@ -0,0 +1,3 @@
+# Not required since we need to have the server statement
+# inside the http statement in nginx.conf for loading environment variables.
+# 'server' config inside nginx.conf inside 'http' config.
```

Comment:
Could we just delete this file then?

Reply:
I'm deleting it and add the info/comment in the file in the readme file